content/docs/SUSHI/configuration/exhaustive-config.yaml

```diff
@@ -124,6 +136,10 @@ groups:
 # file name extension.  Any subproperties that are valid filenames
 # with supported extensions (e.g., .md/.xml) will be treated as
 # sub-pages.
+# For R5 ImplementationGuides, the IG.definition.page element now
+# now supports a source[x] element. SUSHI will generate sourceUrl
+# based on the name if it is not provided. To provide a source,
+# add a key/value pair subproperty with the choice type as the key.
```

Comment:
I think this should maybe say "type slice" instead of "choice type", but I'm actually not sure what the official wording is.
```suggestion
# add a key/value pair subproperty with the type slice as the key.
```

Reply:
I don't know if there is an official word for that -- but I'm fine with this change. It's probably a little more accurate (if people know what type slice means).